core/inc/com/centreon/broker/misc/misc.hh

```diff
@@ -29,6 +29,25 @@ namespace misc {
   std::string temp_path();
   std::list<std::string> split(std::string const& str, char sep);
   uint16_t crc16_ccitt(char const* data, unsigned int data_len);
+
+  class Uuid {
```

Comment:
Would be better to move it into misc/uuid.{cc,hh}

Reply:
done